src/natcap/invest/urban_flood_risk_mitigation.py

```diff
@@ -499,19 +497,17 @@ def _write_summary_vector(
         target_feature.SetGeometry(base_geom_ref.Clone())
         base_geom_ref = None
 
-        if feature_id in runoff_ret_stats:

```

Comment:
Please see my comment in urban cooling; there may still be a reason to do these checks rather than raise a `KeyError`

Reply:
see my response above